core/sodasql/common/time_helper.py

```diff
@@ -0,0 +1,24 @@
+#   Copyright 2021 Soda
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#    http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+import logging
+from datetime import datetime
+
+
+class TimeHelper:
+
+    @staticmethod
+    def datetime_valid(date: str):
+        try:
+            datetime.fromisoformat(date)
+        except ValueError as e:
+            logging.error(f'The scan time is not in ISO8601 format: {str(e)}')
```

Comment:
To avoid confusion about which date we are talking about here, can you include the input value `date`? Or is it maybe already included in the error message?

Reply:
It's part of the error message indeed. 

Example: `ERROR:root:The scan time is not in ISO8601 format: Invalid isoformat string: '2021-04-15T09:00:00+0200'`